### PR TITLE
Add on init fetch token prices

### PR DESCRIPTION
**Pull request by NikolaCBS**
Added a function call to initialize token prices in order to evade errors on a cold launch of the server.

### CODE REVIEW

File: src/modules/tracker/tracker-val.sync.ts
Comment (NikolaCBS) [inline src/modules/tracker/tracker-val.sync.ts:1]: Disregard this comment deletion as it was not pushed by a previous commit.

File: src/modules/tracker/tracker-pswap.sync.ts
Comment (NikolaCBS) [inline src/modules/tracker/tracker-pswap.sync.ts:1]: Disregard this comment deletion as it was not pushed by a previous commit.

Comment (NikolaCBS) [inline src/modules/tracker/tracker-pswap.sync.ts:1]: Disregard this comment deletion as it was not pushed by a previous commit.

File: src/modules/tracker/tracker-supply.sync.ts
Comment (NikolaCBS) [inline src/modules/tracker/tracker-supply.sync.ts:1]: Disregard this comment deletion as it was not pushed by a previous commit.

**Review by DanijelCBS — APPROVED** [review]

**Pull request merged**